scipy/sparse/tests/test_construct.py

```diff
@@ -225,7 +233,8 @@ def test_diags_empty(self):
         x = construct.diags([])
         assert_equal(x.shape, (0, 0))
 
-    def test_identity(self):
+    @pytest.mark.parametrize("identity", [construct.identity, construct.eye_array])
+    def test_identity(self, identity):
         assert_equal(construct.identity(1).toarray(), [[1]])
```

Comment:
Should this be using the `identity` argument instead of `construct.identity`?

Reply:
Yes, indeed it should. :)